src/ledger/transaction.ts

```diff
@@ -112,12 +115,12 @@ function convertError(
 
 function formatResponse(
   options: TransactionOptions,
-  tx: TransactionResponse
+  tx: any
```

Comment:
Are we planning on typing the Transaction Responses eventually?

Reply:
This gets deleted in #1550 anyways.